keyboards/ut472/keymaps/default/keymap.c

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define LT3_TAB LT(3, KC_TAB)
+#define MT_RSFT_ENT MT(MOD_RSFT, KC_ENT)
```

Comment:
You might be able to replace this with `KC_SFTENT`: https://docs.qmk.fm/#/feature_space_cadet

Reply:
fixed